sdks/python/run_mini_py3lint.sh

```diff
@@ -37,4 +37,5 @@ if test $# -gt 0; then
 fi
 
 echo "Running flake8 for module $MODULE:"
-flake8 $MODULE --count --select=E999 --show-source --statistics
+# TODO: Add F821 (undefined names) as soon as that test passes
```

Comment:
Which test passes? Do we have a JIRA tracking this?

Also why F821 is special compared to any other warning to be mentioned in this todo?

Reply:
All tests pass __except__ for F821 which is what makes it special.

Reply:
There are other errors/violation codes that are not enabled including F821: http://flake8.pycqa.org/en/latest/user/error-codes.html

Reply:
Please reread the first sentence of description above which discusses the "showstopper" flake8 issues vs. the "style violations".  I have been focused on the former for the reasons described above.

Reply:
Please reread the first sentence of description above which discusses the "showstopper" flake8 issues vs. the "style violations".  I have been focused on the former for the reasons described above.

Reply:
It is fine to move that sentence to a JIRA, and add a todo referencing that JIRA. People reading that TODO once it is merged would not easily be able to find the PR and understand what is the todo for, and what needs to happen to resolve it.

You can also reference the issues you have noticed with F821 as you mentioned in your other comment in that same JIRA.